fa_scrapper/fa_scrapper.py

```diff
@@ -145,6 +146,29 @@ def get_list_data(user_id, list_id, lang):
             }
 
 
+def get_user_lists(user_id,lang):
+    """Yields all lists from the given user"""
+
+    FA = (
+        FA_ROOT_URL + "userlists.php?user_id={user_id}&p={{}}"
+    ).format(lang=lang, user_id=user_id)
+
+    for page in pages_from(FA):
+        tags = page.find_all(class_=["list-name-wrapper"])
+        for tag in tags:
+            #list_name=tag.text.split('\n')[1]      #Could be used in filename
```

Comment:
I am okay with using list names for the filename if you feel like that would work better (I will leave that up to you). If you want to use it you would need to remove special characters so that we have a valid filename on Windows.